src/time/Convert.cxx

```diff
@@ -57,7 +54,7 @@ LocalTime(std::chrono::system_clock::time_point tp)
 {
 	const time_t t = std::chrono::system_clock::to_time_t(tp);
 #ifdef _WIN32
-	const struct tm *tm = localtime(&t);
+	const struct tm *tm = std::localtime(&t);
 #else
 	struct tm buffer, *tm = localtime_r(&t, &buffer);
```

Comment:
Which header provides `localtime_r` now?

Reply:
I believe ctime includes time.h. I'll try to find a reference from the standard.